src/build/lift_functions/IR.zig

```diff
@@ -9,7 +9,7 @@ const Problem = problem.Problem;
 const FieldName = collections.FieldName;
 const StringLiteral = collections.StringLiteral;
 
-pub const IR = @This();
+const Self = @This();
```

Comment:
Would be good to keep the naming convention of `IR` for this.

Reply:
I can see the convention of Self, so this might actually be the right direction, but all `@This()`s being named `Self` leads to naming conflicts when the same file has multiple structs. So I don't know if it's *clearly* the right choice, even if its advantages are clear

Reply:
Okay, good pattern!

Reply:
Cool! I update the references to `This()` in each stage to follow this pattern as well, per our Zulip DMs